service/notifications.yml

```diff
@@ -79,8 +79,8 @@ alkemio:
   endpoint: ${ALKEMIO_SERVER_ENDPOINT}:http://localhost:3000/api/private/non-interactive/graphql
   webclient_endpoint: ${ALKEMIO_WEBCLIENT_ENDPOINT}:http://localhost:3000
   service_account:
-    username: ${SERVICE_ACCOUNT_USERNAME}:notifications@alkem.io
-    password: ${SERVICE_ACCOUNT_PASSWORD}:change-me-now
+    username: ${SERVICE_ACCOUNT_USERNAME}:admin@alkem.io
+    password: ${SERVICE_ACCOUNT_PASSWORD}:Fin1.tan
```

Comment:
This won't work for local setup. Why are we changing these? I assume it was committed by accident and was for testing.

Reply:
Yes by accident - I will destroy that branch + re-create